aspnetcore/signalr/hubs.md

```diff
@@ -23,6 +23,9 @@ To register the services required by SignalR hubs, call <xref:Microsoft.Extensio
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Program.cs" id="snippet_AddSignalR" highlight="4":::
 
+> [!NOTE]
+> With the release of .NET Core 3.0, many ASP.NET Core assemblies, including those of SignalR, are no longer published to NuGet as packages. Instead, the assemblies are included in the Microsoft.AspNetCore.App shared framework, which is installed with the .NET Core SDK and runtime installers. See [Use the ASP.NET Core shared framework](xref:fundamentals/target-aspnetcore#use-the-aspnet-core-shared-framework) for more information.
+
```

Comment:
Do we really want to have this first thing at the top of the article? The Note is too much bling.

```suggestion
SignalR no longer depends on getting many assemblies from NuGet, those assemblies are contained in the `Microsoft.AspNetCore.App` shared framework. For more information, see [Microsoft.AspNetCore.App replaces many NuGet packages](xref:the/right/migration/doc#H2).
```

I recommend putting this at the bottom of the file. Experienced SignalR dev's migrating aren't going to look in this doc, they'll look in the migration doc. OK to put a link at the bottom tho

Reply:
I agree.

Reply:
@Rick-Anderson, I'll put it at the end of the topics as you ask but drop it back into a note.  That way it doesn't not look like it is part of the context of whatever the last section is, and a dedicated new section does not make sense of course.